### DISCUSSION TITLE
File Explorer changes are't mirrored in Spacedrive

### DISCUSSION THREAD

**Discussion by Pezzz03** [spacedriveapp/spacedrive#2603] [Community Support]
I'm not entirely sure how to describe the problem, but when I first started using spacedrive it would mirror changes that I made in file explorer. If there were new folders created or if things moved around (in file explorer) then spacedrive would be updated to reflect these changes and would be exactly the same as the file explorer. Now, it seems to have stopped and it doesnt update the changes made. I have tried re-scanning the locations and re-indexing folders but nothing seems to work. I am not sure whether this is a bug with Spacedrive or if I am doing something wrong. If it helps troubleshoot the problem i am using spacedrive v0.3.1 on windows 11

**Comment by augviolet** [comment]
I also found this problem, so far I tested it seems that such a problem occurs in the case that if there is a spacedrive started, then the changes in the local file explorer will be updated in real time to space, but if it is a change in the local explorer after space is closed, it will be the case you mentioned
